docs/manual/scala/guide/cluster/ReadSideSlick.md

```diff
@@ -102,3 +106,9 @@ This can then be registered with the builder using `setEventHandler`:
 Once you have finished registering all your event handlers, you can invoke the `build` method and return the built handler:
 
 @[build](code/docs/home/scaladsl/persistence/SlickBlogEventProcessor.scala)
+
+## Application Loader
+
+The Lagom Application loader needs to to be configured for Slick persistence. This can be done by extending the `SlickPersistentComponents` class like so:
```

Comment:
I think we could replace `extending` with `mixing in`. `SlickPersistentComponents` is a trait we add to the _cake_ of the application.

Reply:
Makes sense @ignasi35 -- I will update it. I will also attempt to complete doc compilation before I submit another commit.